components/charts/LineData/index.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+
+const LineData = ({
+  data = [],
+  xDomain,
+  xScale,
+  yScale,
+  ticks,
+  labelledById,
+}) => {
+  const interval = (xDomain[1] - xDomain[0]) / (ticks - 1);
+
+  const points = data.reduce((prev, curr, i) => {
+    const nextPoint = `${prev} ${xScale(xDomain[0] + interval * i)},${yScale(
+      curr
+    )}`;
+
+    return nextPoint;
+  }, "");
+
+  return <Styled.Line points={points} aria-labelledby={labelledById} />;
```

Comment:
how can the line data be made accessible? It does not have the "list" format of bar graphs where individual datapoints can be read out. Points could be assigned to the line and be invisible by default but included in the accessibility tree.